src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -91,6 +91,10 @@
         // Use on a card that remembers a memory, which is mechanic specific to the Wanderer character.
         @SpireEnum(name = "REMEMBER_MEMORY")
         public static AbstractCard.CardTags REMEMBER_MEMORY;
+
+        // Cards in the Deck of Toils should be
+        @SpireEnum(name = "DECK_OF_TOILS")
```

Comment:
I think it's "trials", not "toils"

Reply:
I'll check in Discord. it's definitely inconsistent, Book of Toils relic description indicates "toils", bottom section indicates "trials"

Reply:
trials it is!